packages/sdk-browser/README.md

```diff
@@ -12,27 +12,126 @@ Turnkey API documentation lives here: https://docs.turnkey.com.
+by passing in `https://auth.turnkey.com` as the `iframeUrl`.
+
+```typescript
+const iframeClient = await turnkey.iframeClient({
+  // The container element that will host the iframe
+  iframeContainer: document.getElementById("<iframe container id>"),
+  iframeUrl: "https://auth.turnkey.com",
+});
+
+const response = await iframeClient.injectCredentialBundle(
+  "<Credential from Email>"
+);
+if (response) {
+  await iframeClient.getWallets();
+}
+```
+
+##### IFrame URLs:
+
+| Flow                                                                                  | URL                                                  |
+| ------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| [Email Auth](https://docs.turnkey.com/embedded-wallets/sub-organization-auth)         | [auth.turnkey.com](https://auth.turnkey.com)         |
+| [Email Recovery](https://docs.turnkey.com/embedded-wallets/sub-organization-recovery) | [recovery.turnkey.com](https://recovery.turnkey.com) |
+| [Import Wallet](https://docs.turnkey.com/features/import-wallets)                     | [import.turnkey.com](https://import.turnkey.com)     |
+| [Export Wallet](https://docs.turnkey.com/features/export-wallets)                     | [export.turnkey.com](https://export.turnkey.com)     |
+
+#### Wallet
+
+The Wallet client is designed for using your Solana or EVM wallet to stamp and approve activity requests for Turnkey's API.
+This stamping process leverages the wallet's signature to authenticate requests.
+
+The example below showcases how to use an injected Ethereum wallet to stamp requests to Turnkey's API.
+The user will be prompted to sign a message containing the activity request payload to be sent to Turnkey.
+
 ```typescript
 import {
-  TurnkeyBrowserSDK,
-  TurnkeySDKBrowserConfig,
-  TurnkeySDKBrowserClient,
-} from "@turnkey/sdk-browser";
+  createWalletClient,
+  custom,
+  recoverPublicKey,
+  hashMessage,
+} from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { mainnet } from "viem/chains";
+
+import { WalletStamper } from "@turnkey/wallet-stamper";
+
+// Create a new wallet client with a JSON-RPC account from the injected provider
+const walletClient = createWalletClient({
+  chain: mainnet,
+  transport: custom(window.ethereum!),
+});
 
-// This config contains parameters including base URLs, iframe URLs, org ID, and rp ID (relying party ID for WebAuthn)
-import turnkeyConfig from "./turnkey.json";
+const signMessage = async (message: string) => {
+  const account = walletClient.account.address;
+  return walletClient.signMessage({
+    account,
+    message,
+  });
+};
 
-// Use the config to instantiate a Turnkey Client
-const turnkeyClient = new TurnkeyBrowserSDK(turnkeyConfig);
+const getPublicKey = async () => {
+  // Required to recover the secp256k1 public key from the signature
```

Comment:
as the `arbitraryMessage` variabole name indicates, we can explicitly add that the content of the message doesn't matter